src/omnisharp/extension.ts

```diff
@@ -88,6 +88,25 @@ export function activate(context: vscode.ExtensionContext, reporter: TelemetryRe
         }));
     }
 
+    // After server is started (and projects are loaded), check to see if there are
+    // any project.json projects. If so, notify the user about migration.
+    disposables.push(server.onServerStart(() => {
+        utils.requestWorkspaceInformation(server)
+            .then(workspaceInfo => {
+                if (workspaceInfo.DotNet && workspaceInfo.DotNet.Projects.length > 0) {
```

Comment:
Will this fire for the old PCL projects that use project.json for dependencies but still have a .csproj? Do you want to deprecate those as well?